build/tasks.ts

```diff
@@ -255,4 +202,24 @@ register("package", async (runner, releaseTag) => {
 		: runner.execute("zip", ["-r", `${archiveName}.zip`, `${archiveName}`]);
 });
 
+/**
+ * If we're in the CI and the VS Code version is the same,
+ * then we don't need to rebuild.
+ */
+const ifCiAndVsc = (vscName: "vscode" | "vscode-default-extensions"): boolean => {
```

Comment:
Should this be used somewhere?

Reply:
Meant for the old CI stuff. Will remove!